addons/portal/wizard/portal_share.py

```diff
@@ -72,7 +72,7 @@ def _send_public_link(self, note, partners=None):
             self.resource_ref.message_post_with_view(template,
                 values={'partner': partner, 'note': self.note, 'record': self.resource_ref,
                         'share_link': share_link},
-                subject=_("You are invited to access %s", self.resource_ref.display_name),
+                subject=_("Invitation to access %s", self.resource_ref.display_name),
```

Comment:
I know LNA struggled with the wording of portal / templates stuff, did you check with him of he feels things :D ?

Reply:
No indeed, I'll ask my boss to do so :o)

Reply:
LNA is ok with it.